### DISCUSSION TITLE
Is it possible to have Argo Workflows post status updates to Github pull requests?

### DISCUSSION THREAD

**Discussion by omarkalloush** [argoproj/argo-workflows#9107] [Q&A] [answered]
Is there a plugin or a way to have Argo workflows post these status updates to Github pull requests?

![Status Updates](https://raw.githubusercontent.com/cloudposse/github-status-updater/master/images/github-status-check-pending.png)

**Comment by terrytangyuan** [comment, marked as answer]
Check out this: https://github.com/sendible-labs/ci-github-notifier

More resources: https://github.com/terrytangyuan/awesome-argo
